src/openrct2/windows/view_clipping.c

```diff
@@ -0,0 +1,267 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include "../config.h"
+#include "../interface/themes.h"
+#include "../interface/widget.h"
+#include "../interface/window.h"
+#include "../interface/viewport.h"
+#include "../localisation/localisation.h"
+#include "../paint/paint.h"
+#include "../rct2.h"
+
+enum WINDOW_VIEW_CLIPPING_WIDGET_IDX {
+	WIDX_BACKGROUND,
+	WIDX_TITLE,
+	WIDX_CLOSE,
+	WIDX_CLIP_HEIGHT_CHECKBOX,
+	WIDX_CLIP_HEIGHT_VALUE,
+	WIDX_CLIP_HEIGHT_INCREASE,
+	WIDX_CLIP_HEIGHT_DECREASE,
+	WIDX_CLIP_HEIGHT_SLIDER
+};
+
+#pragma region Widgets
+
+#define WW 160
+#define WH 70
+
+rct_widget window_view_clipping_widgets[] = {
+	{ WWT_FRAME,           0,  0,       WW - 1, 0,      WH - 1, STR_NONE,                        STR_NONE }, // panel / background
+	{ WWT_CAPTION,         0,  1,       WW - 2, 1,      14,     STR_VIEW_CLIPPING,               STR_WINDOW_TITLE_TIP }, // title bar
+	{ WWT_CLOSEBOX,        0,  WW - 13, WW - 3, 2,      13,     STR_CLOSE_X,                     STR_CLOSE_WINDOW_TIP }, // close x button
+	{ WWT_CHECKBOX,        1,  11,      149,    19,     29,     STR_VIEW_CLIPPING_HEIGHT_ENABLE, STR_VIEW_CLIPPING_HEIGHT_ENABLE_TIP }, // clip height enable/disable check box
```

Comment:
Ah.  That's helpful.  Thanks.
I realised that this indicated some sort of grouping, but hadn't worked out it was related to the colours.
Now I will reduce the theme to one colour as suggested.